### PR TITLE
Preselect organization's default team in dataset upload view

### LINKED ISSUE #8517: In Basic Plan, Members can see no datasets. Consider auto-setting default team in this case
Normally, datasets are not shared with Members unless explicitly selected with team sharing. However, in Basic plan, teams aren’t available. In this case, every orga member should see all datasets. (Possibly by auto-setting the default team for new datasets)

### PR DESCRIPTION
**Pull request by knollengewaechs**
### Steps to test:
- Go to the dashboard an upload a dataset. You should see the default team being selected. Thus for orgas in the basic plan, all DS should be accessible for the default team. 
- Check that the team is really set

### TODOs:
- [x] refresh snapshots

### Issues:
- fixes #8517 

------
(Please delete unneeded items, merge only when none are left open)
- [x] Added changelog entry (create a `$PR_NUMBER.md` file in `unreleased_changes` or use `./tools/create-changelog-entry.py`)
- [ ] Added migration guide entry if applicable (edit the same file as for the changelog)
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [ ] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

File: frontend/javascripts/test/fixtures/dataset_server_object.ts [hunk 1]
```diff
@@ -90,13 +90,15 @@ function createDataset(dataLayers: Array<APIColorLayer | APISegmentationLayer>):
       {
         id: "5b1e45f9a00000a000abc2c3",
         name: "Connectomics department",
+        isOrganizationTeam: true,
         organization: "Connectomics department",
       },
```
[the remark this answers is not in the document]
Comment (hotzenklotz) [reply frontend/javascripts/test/fixtures/dataset_server_object.ts:95]: @knollengewaechs That is a valid concern but after double-checking the other mention fixture files, I disocvered that they only had empty `team` objects.
